app/assets/stylesheets/main.scss

```diff
@@ -62,6 +63,40 @@ body {
   box-shadow: inset 0 -5px 0 #b4e7f8;
 }
 
+/* start of styles for cards on index.html.erb page */
+.ice-index {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: scroll;
+  height: 350px;
```

Comment:
This should be set to auto for mobile devices with a media query.
``` CSS
@media (max-width 468px) {
height: auto;
}
```
Start with the `468px